src/resources/language/resource.language.en_gb/strings.po

```diff
@@ -375,3 +375,11 @@ msgstr ""
 msgctxt "#32091"
 msgid "3D"
 msgstr ""
+
+msgctxt "#33001"
+msgid "HLS2/4 Stream-Manifest"
+msgstr ""
+
+msgctxt "#33002"
```

Comment:
I believe this is from another PR.

Reply:
"I'm not sure if the plugin should fix incorrect data that comes from the API. We could keep it as a temporary workaround. But I think we should address this to kinopub maintainers. What do you think?"

In general, you're right. This is a workaround, not a fix. But I found out that many streams are corrupted in this way. And when a user tries to watch such a corrupted stream the Kodi will often select the wrong codec (H.264 instead of H.265 or H.265 instead of H.264) and the stream is not playable. Fixing the source problem could take a long time and in the meantime the stream is unplayable.

IMHO for such use cases the add-on can support the user and *try* to fix the stream (which is only a workaround).

Reply:
The strings from another PR are removed.
Additionally i added handling the relative urls for subtitles (this is needed because at time the requesting subtitile the proxy is already shutdown)